xml/System/Guid.xml

```diff
@@ -1260,10 +1313,55 @@
+        <param name="value">An object to compare to this instance.</param>
+        <summary>Compares this instance to a specified <see cref="T:System.Guid" /> object and returns an indication of their relative values.</summary>
+        <returns>A signed number indicating the relative values of this instance and <paramref name="value" />.
+          <list type="table">
+            <listheader>
+              <term>Return value</term>
+              <description>Description</description>
+            </listheader>
+            <item>
+              <term>A negative integer</term>
+              <description>This instance is less than <paramref name="value" />.</description>
+            </item>
+            <item>
+              <term>Zero</term>
+              <description>This instance is equal to <paramref name="value" />.</description>
+            </item>
+            <item>
+              <term>A positive integer</term>
+              <description>This instance is greater than <paramref name="value" />.</description>
+            </item>
+          </list>
+        </returns>
+        <remarks>
+          <format type="text/markdown"><![CDATA[
+
+## Remarks
+
+The <xref:System.Guid.CompareTo%2A> method compares the GUIDs as if they were values provided to the <xref:System.Guid.%23ctor%28System.Int32%2CSystem.Int16%2CSystem.Int16%2CSystem.Byte%5B%5D%29> constructor, as follows:
+
+-   It compares the <xref:System.UInt32> values, and returns a result if they are unequal. If they are equal, it performs the next comparison.
+-   It compares the first <xref:System.UInt16> values, and returns a result if they are unequal. If they are equal, it performs the next comparison.
+-   It compares the second <xref:System.UInt16> values, and returns a result if they are unequal. If they are equal, it performs the next comparison.
+-   If performs a byte-by-byte comparison of the next eight <xref:System.Byte> values. When it encounters the first unequal pair, it returns the result. Otherwise, it returns 0 to indicate that the two <xref:System.Guid> values are equal.
+
+Note that the final eight bytes appear in the string representation of a <xref:System.Guid> in reverse order, from low byte to high byte. For example, in the string representation of the <xref:System.Guid> value "01e75c83-c6f5-4192-b57e-7427cec5560d", the final eight bytes are "b57e-7427cec5560d." In other words, the final eight bytes are compared on a byte-by-byte basis from left to right starting with 0xb5.
+
+If two GUIDs have equal values for a component, the method compares the next component. When it finds a component whose values are unequal, it returns the result.
+
+This method implements the <xref:System.IComparable%601?displayProperty=nameWithType> interface and performs slightly better than the <xref:System.Guid.CompareTo%2A?displayProperty=nameWithType> method because it does not have to convert the `value` parameter to a <xref:System.Guid> value.
+
+## Examples
```

Comment:
The example should be removed, since it calls Guid.CompareTo instead of Guid.IComparable.CompareTo.